<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanWindow</class>
 <widget class="QDialog" name="ScanWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    border-image:url(:/files/cam_bg.jpg);
    width: 100px;
    border: none;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="lUsed">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>48</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: red;
}</string>
     </property>
     <property name="text">
      <string>Код уже использован</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pbSendCode">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="leCode"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
